### DISCUSSION TITLE
Feature request: Mapbox Maps SDK for macOS support

### DISCUSSION THREAD

**Discussion by gorbypark** [rnmapbox/maps#1105] [Ideas]
Microsoft and Facebook have been hard at work porting React Native to Windows and macOS.  Although Mapbox doesn't appear to have a Windows SDK, they do have a macOS SDK available.  From my limited investigation, it appears the macOS and iOS SDKs are fairly similar, so in theory a lot of work could be re-used.  It would be great to be able to use this wonderful component when working on desktop macOS apps.

**Comment by ferdicus** [comment]
Sounds good 👍🏿 
Looking forward to the PRs

**Comment by systemlevel** [comment]
With the new push from Apple to get iOS apps onto MacOS with their own Silicon, etc. this might be a big deal for many. @gorbypark How good are you with the macOS SDK?

**Comment by RobertSasak** [comment]
As discussed at Gitter. https://gitter.im/react-native-mapbox-gl/Lobby?at=5fabd8212a60f731f74b1e34

It is not possible to target MacOs as underlying Mapbox iOS is still using EAGLContext which is deprecated. mapbox/mapbox-gl-native-ios#117

Any other suggestions?

**Comment by mfazekas** [comment]
I'm closing this as with v10, mapbox should no longer use OpenGL
